inst/rmarkdown/templates/pihph/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+title: This is a title and this is too
+subtitle: A subtitle goes on another line
+author:
+- name: Author's name
+  affiliation: Author's affiliation
+  email: e-mail@uni.edu
+- name: Second author's name (if there is one)
+  affiliation: Second author's affiliation
+output: rticles::pihph_article
+gloss: expex # feel free to use your preferred package for the examples
+# FONTS
+# If default-fonts is true, alt-fonts should be set to false, and vice versa.
+default-fonts: false # set to true if you want to use the defaults fonts
+# The default fonts are: Cambria, Calibri, and Lekton
+# Lekton can be found at: https://www.fontsquirrel.com/fonts/lekton
+alt-fonts: false # set to true if you want to use alternative fonts
+# The alternative fonts are: Caladea, Carlito, and Lekton
+# Caladea can be found at: https://fontlibrary.org/en/font/caladea
+# Carlito can be found at: https://fontlibrary.org/en/font/carlito
+# Lekton can be found at: https://www.fontsquirrel.com/fonts/lekton
+#
+# If any of these fonts are not available on your system, feel free to
+# change them here, but of course the final layout will then differ.
+# Uncomment the following lines and set the chosen fonts:
+# mainfont: Charis SIL
+# licensefont: Arial
+# logofont: Arial
+#
+# Finally, if you'd rather use the default XeLaTeX fonts, just leave
+# both default-fonts and alt-fonts set to false and don't specify mainfont,
+# licensefont, nor logofont.
+abstract: |
+ This is the template for \LaTeX{} users; we also have a Word template available, so feel free to use that if you're not into \LaTeX. All submissions to PiHPh should aim to follow the formatting set out in this template precisely, as there is no separate typesetting phase. Most styles in this template have been set up to insert the necessary white space where needed. You should use the styles that are provided in this template consistently to ensure that your submission is quickly processed. Do not change the margins or header/footer properties, including the material on this first page (such as volume, page numbers and DOI), which we will update when your submission is ready. Make sure to look at the user customization part of the preamble to this \texttt{.tex} source to change titles, running headers, bibliography etc. All the packages used here should be available in a recent version of \TeX Live. All articles should include an abstract in this position, of no more than 200 words.
+acknowledgements: Include any acknowledgements in an unnumbered section here, rather than in a footnote early in the paper.
+supplement: If you would like to post any data sets, statistical data, scripts or similar material that links to your article, you can include a description of it all at the end of the paper.
+---
```

Comment:
You have more variables in the template that here in the YAML header. 

* Should we keep them hidden ? 
* Document them as comment in the YAML header ? 
* Document them as a section in the help of the function ? 

Other template document option variable by commenting them (as you did for fonts above). 

I think we should at least add an example for bibliography. This would allow to test that it works (which currently don't)